clients/erigon/config-alloy.template

```diff
@@ -0,0 +1,206 @@
+		action        = "keep"
+	}
+}
+
+prometheus.remote_write "default" {
+	external_labels = {
+		cluster = "prod-nodes",
+	}
+
+	endpoint {
+		name = "prom-ep"
+		url  = "https://prometheus-prod-24-prod-eu-west-2.grafana.net/api/prom/push"
+
+		basic_auth {
+			username = "1322871"
+			password = "${GRAFANA_PROM_BASICAUTH}"
+		}
```

Comment:
This needs to be updated to the new metrics endpoint.  You can see the node-setup script for the new values.